src/test/java/ladder/domain/LineTest.java

```diff
@@ -0,0 +1,50 @@
+package ladder.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+class LineTest {
+    private Line line;
+
+    @BeforeEach
+    void setUp() {
+        line = new Line();
+    }
+
+    @Test
+    @DisplayName("addBars()은 랜덤 값이 false로 나올 때, 생성되는 bars")
+    void test_1() {
+        // when
+        line.addBars(4, () -> false);
+
+        // then
+        Assertions.assertThat(line.getBars()).isEqualTo(List.of(BarTest.FALSE, BarTest.FALSE, BarTest.FALSE, BarTest.FALSE));
```

Comment:
```suggestion
        assertThat(line.getBars())
            .contains(BarTest.FALSE, BarTest.FALSE, BarTest.FALSE, BarTest.FALSE);
```
테스트에서 자주 사용하는 메서드는 import static으로 올려버리면 더 깔끔하게 테스트를 작성할 수 있어요.
그리고 assert 메서드 체이닝 부분에서 개행을 해주시면 가독성이 더 좋아집니다ㅎㅎ
마지막으로 collection에 어떤 값이 있는지는 contains라는 메서드를 한번 찾아보고 사용해보면 좋을 것 같아요.

Reply:
테스트 관련 메서드에 대한 공부가 부족했네요. 학습을 더 해야겠네요. 좋은 방법 알려주셔서 감사합니다.